wolox-books/src/app/pipes/filter-book.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Book } from '../screens/book/book';
+
+@Pipe({
+  name: 'filterBook'
+})
+export class FilterBookPipe implements PipeTransform {
+
+  transform(books: Book[], search: string): any {
+    if(!!search && search.trim() != "") {
```

Comment:
you can remove this if

Reply:
I can't, if I remove the pipe filtered by an empty string, and wont show anything